conf/evolutions/default/13.sql

```diff
@@ -6,4 +6,4 @@ ALTER TABLE "MenuPerDayPerPerson"
 # --- !Downs
 
 ALTER TABLE "MenuPerDayPerPerson"
-  DROP COLUMN "timestamp";
+  DROP COLUMN "timestamp";
```

Comment:
Can you undo this change? I have the idea that modifying scripts that were already applied is problematic